packages/teams-js/src/public/runtime.ts

```diff
@@ -100,7 +100,8 @@ export const teamsRuntimeConfig: IRuntime = {
       bot: {},
       update: {},
     },
-    logs: {},
+    // files: {},
```

Comment:
Kept this for future reference when files will be supported.

Reply:
I'm not sure we need this. We don't know what shape files will be in when it's ready for capability support, so I think we should just remove this.